scripts/gc.sh

```diff
@@ -30,50 +30,58 @@ mkdir -p "$DIRNAME"
 mkdir -p "$DIRNAME/src"
 mkdir -p "$DIRNAME/__tests__"
 
-cat > $DIRNAME/src/index.vue <<EOF
+cat > $DIRNAME/src/$INPUT_NAME.vue <<EOF
 <template>
   <div>
-    <slot></slot>
+    <slot />
   </div>
 </template>
-<script lang='ts'>
-import { defineComponent } from 'vue'
-export default defineComponent({
-  name: 'El${NAME}',
-  props: { },
-  setup(props) {
-    // init here
-  },
+<script lang="ts" setup>
+import { ${INPUT_NAME}Props } from './$INPUT_NAME'
+
+defineOptions({
+  name: 'El$NAME',
 })
+
+const props = defineProps(${INPUT_NAME}Props)
+
+// init here
 </script>
-<style>
-</style>
+<style></style>
+EOF
+
+cat > $DIRNAME/src/$INPUT_NAME.ts <<EOF
+import { buildProps } from '@element-plus/utils'
+import type { ExtractPropTypes } from 'vue'
```

Comment:
```shell
cat > $DIRNAME/src/$INPUT_NAME.ts <<EOF
 import { buildProps } from '@element-plus/utils'
 import type { ExtractPropTypes } from 'vue'
```
In this case, the generated file will also have a space, and it will also be formatted and indented when saved.

Reply:
I mean insert a **blank line** between `import...` and `import type...`, will this have formatting issues?

Reply:
I understand.😋